### DISCUSSION TITLE
Issue with Isoform ID

### DISCUSSION THREAD

**Discussion by daacarri** [egeulgen/pathfindR#104] [Q&A]
Hi again, 

I noticed that genes assigned to the significant pathways arent always isoform specific. For example, in my data POU5F1 was listed in the pluripotency maintenance pathway (significant). However, in the DE data, POU5F1 is not significant or changes, but the pseudogene POU5F1p4 is. Is this a ID mapping error? I suspect the DE data has a more comprehensive gene ID list, whereas pathfindR draws only limited gene IDs from the different path lists etc. Will pathfindR try to best match the closest gene ID? 

Thanks again,

**Comment by egeulgen** [comment]
pathfindrR maps the genes onto the interaction network and while doing that, checks the alias symbols for symbols that are missing in the network (this only works for homo sapiens gene symbols).
The built-in data for gene sets/interaction networks contain gene symbols so cannot be isoform-specific, that is correct. However, you have the option to use custom gene sets/interaction networks. Using custom data, you can use practically use any form of gene IDs (entrez ID, ensembl gene IDs, transcript IDs, etc.) as long as they are consistent between the protein-protein interaction network, gene sets and input data.
